src/Features/Core/Portable/CodeRefactorings/SyncNamespace/SyncNamespaceCodeRefactoringProvider.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.CodeRefactorings.SyncNamespace
+{
+    [ExportCodeRefactoringProvider(LanguageNames.CSharp, Name = PredefinedCodeRefactoringProviderNames.SyncNamespace), Shared]
```

Comment:
not exported for VB.

Reply:
VB provider is not implemented. The purpose of VB logic in the PR is to handles the case when references to the types declared in changed namespace are in VB projects. The refactoring can only trigger in C# document.

This is partially because of the difference between default namespace and root namespace in C# and VB. I need to have a better understanding on VB side before implementing that.